netdata-installer.sh

```diff
@@ -956,6 +963,81 @@ bundle_ebpf_co_re() {
 
 bundle_ebpf_co_re
 
+# -----------------------------------------------------------------------------
+build_fluentbit() {
+  local env_cmd=''
+
```

Comment:
please, take a look in another shellcheck here.

Reply:
Fixed.